scripts/2.TCGA-process.py

```diff
@@ -232,37 +244,44 @@
 expr_df.shape
 
 
-# In[20]:
+# In[21]:
 
 # Number of patients represented in the expression dataset
 clinmat_df.query("sample_id in @expr_df.index").patient_id.nunique()
 
 
-# In[21]:
+# In[22]:
 
 # Peak at the data matrix
 expr_df.iloc[:5, :5]
 
 
+# In[23]:
+
+# Save complete expression matrix
+path = os.path.join('data', 'complete', 'expression-matrix.tsv.bz2')
+expr_df.to_csv(path, sep='\t', float_format='%.3g', compression='bz2')
```

Comment:
Do you know how much the `float_format` changes things here? Does it at all? I have slight concerns that someone using the unformated Xena data may get different results than if they were to use our formatted data